Arrowgene.Ddon.GameServer/Party/PartyQuestState.cs

```diff
@@ -655,9 +655,9 @@ private void SendWalletRewards(DdonGameServer server, GameClient client, Quest q
             }
         }
 
-        public void UpdatePriorityQuestList(DdonGameServer server, PartyGroup party)
+        public void UpdatePriorityQuestList(DdonGameServer server, GameClient requestingClient, PartyGroup party)
```

Comment:
You can probably simplify this by removing the party argument and accessing it by doing requestingClient.Party, i don't think you'd want a case where the provided party argument isn't the requesting client's party